datasets/aeslc/README.md

```diff
@@ -3,6 +3,17 @@ languages:
 - en
 paperswithcode_id: aeslc
 pretty_name: AESLC
+task_categories:
+- text-classification
+task_ids:
+- "summarization"
+- "summarization-other-conversations-summarization"
+- "other-other-query-based-multi-document-summarization"
+- "summarization-other-aspect-based-summarization"
+- "summarization--other-headline-generation"
+task_categories:
+- "summarization"
+- "dialogue-system"
```

Comment:
I think dialogue system and classification are not relevant since I don't think this dataset has been used for such tasks in research

I suggest we simply keep the summarization tags, what do you think ?

```suggestion
task_categories:
- summarization
task_ids:
- summarization--other-email-headline-generation
- summarization-other-conversations-summarization
- summarization-other-multi-document-summarization
- summarization-other-aspect-based-summarization
```